features/step_definitions/admin_steps.rb

```diff
@@ -0,0 +1,130 @@
+Given(/^I have an admin account$/) do
+  @admin = AdminUser.create(email: "admin@example.com", password: "password", 
+    password_confirmation: "password")
+  # byebug
+end
+
+When(/^I visit the admin panel url$/) do
+  visit("/admin/login")
+end
+
+When(/^I enter my admin email$/) do
+  fill_in "Email*", with: "admin@example.com"
+end
+
+When(/^I enter my admin password$/) do
+  fill_in "Password*", with: "password"
+end
+
+When(/^I click submit$/) do
+  click_button("Login")
+end
+
+Then(/^I see the admin panel$/) do
+  expect(page).to have_content("Dashboard")
+end
+
+Given(/^I do not have an admin account$/) do
+  AdminUser.delete_all
+end
+
+Then(/^I see a flash notification that tell me that my email does not exist in the system$/) do
+  expect(page).to have_content("Invalid email or password")
```

Comment:
I think this should be more specific. You're simply looking for the text. You can also key on the html class, to make sure it's actually in a flash notification.